powerauth-restful-server-javaee/src/main/java/io/getlime/security/powerauth/app/rest/api/javaee/controller/v2/EncryptedDataExchangeController.java

```diff
@@ -84,6 +84,7 @@
         try {
              encryptor = encryptorFactory.buildNonPersonalizedEncryptor(request);
         } catch (RemoteException ex) {
+            logger.warn("Remote communication failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.